tempesta_fw/t/functional/tests/helpers/apache.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+import subprocess
+__author__ = 'NatSys Lab'
+__copyright__ = 'Copyright (C) 2016 NatSys Lab. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+def start():
+	subprocess.call("service apache2 start", shell = True)
```

Comment:
What if Apache HTTP isn't installed on the system? We shall correctly process the case: show the error message to a user and finalize the test.